boot.py

```diff
@@ -106,6 +111,10 @@ def plugin_loaded() -> None:
     client_configs.update_configs()
     for window in sublime.windows():
         windows.lookup(window)
+        ensure_server_panel(window)
+        ensure_references_panel(window)
+        ensure_diagnostics_panel(window)
+        ensure_rename_panel(window)
```

Comment:
Think this should happen in `WindowManager` constructor. Should then also handle more cases.

Reply:
Actually I'm thinking that references and rename panels shouldn't be created initially.

Reply:
The diagnostics panel is already created from there (`WindowManager._update_panel_main_thread`).